### PR TITLE
Marker and Card click event

### PR DESCRIPTION
**Pull request by dexterastin**
When a card or marker is clicked, it focuses on that marker.

### CODE REVIEW

**Comment by dexterastin** [comment]
If swap card, focus map is not work. :(

File: front-end/index.html [hunk 1]
```diff
@@ -12,124 +12,58 @@
...
     <div class="carousel">
-        <div class="carousel-item">
+        <div class="carousel-item" id="0">
             <div class="card">
                 <div class="card-image">
                     <img src="images/300x200.png">
                 </div>
                 <div class="card-content">
                     This is a content
+                    A
```
Comment (shberry) [inline front-end/index.html:22]: Can you add a TODO for replacing this placeholder with actual data?

File: front-end/index.html [hunk 2]
```diff
@@ -12,124 +12,58 @@
...
                 </div>
             </div>
         </div>
-        <div class="carousel-item">
+        <div class="carousel-item" id="1">
             <div class="card">
```
Comment (shberry) [inline front-end/index.html:28]: how about card-container? Since this contains image and content

File: front-end/js/script.js [hunk 3]
```diff
@@ -1,27 +1,43 @@
...
     var labels = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
-    // Add some markers to the map.
-    // Note: The code uses the JavaScript Array.prototype.map() method to
-    // create an array of markers based on a given "locations" array.
-    // The map() method here has nothing to do with the Google Maps API.
-    var markers = locations.map(function (location, i) {
-        return new google.maps.Marker({
-            position: location,
+
+    var latlngbounds = new google.maps.LatLngBounds();
```
Comment (shberry) [inline front-end/js/script.js]: Let's follow the camel case conversion. latlngbounds should be latLngBounds. Same below.

File: front-end/js/script.js [hunk 4]
```diff
@@ -1,27 +1,43 @@
...
+
+    for (i = 0; i < locations.length; i++) {
```
Comment (shberry) [inline front-end/js/script.js]: how about 'location' instead of 'i'? To be more meaningful.

File: front-end/js/script.js [hunk 5]
```diff
@@ -1,27 +1,43 @@
...
+        var data = locations[i];
+        var myLatlng = new google.maps.LatLng(data['lat'], data['lng']);
```
Comment (shberry) [inline front-end/js/script.js]: Good job with camel case :)

File: front-end/js/script.js [hunk 6]
```diff
@@ -1,27 +1,43 @@
 function initMap() {
     var map = new google.maps.Map(document.getElementById('map'), {
-        zoom: 7,
-        center: locations[0]
+        zoom: 7
     });
-    // Create an array of alphabetical characters used to label the markers.
+
```
Comment (shberry) [inline front-end/js/script.js:5]: remove extra line?

**Review by shberry — COMMENTED** [review]
Cool! PTAL

File: front-end/js/script.js (hunk 5 above)
Comment (dexterastin) [reply front-end/js/script.js]: cool! thank you! :)

File: front-end/js/script.js (hunk 4 above)
Comment (dexterastin) [reply front-end/js/script.js]: Modified it

File: front-end/js/script.js (hunk 3 above)
Comment (dexterastin) [reply front-end/js/script.js]: oh, thank you.

File: front-end/index.html (hunk 2 above)
Comment (dexterastin) [reply front-end/index.html:28]: Umm... However, `carousel-item` is a class supported by `materialize`.

**Pull request merged**